bin/hrx

```diff
@@ -34,12 +41,74 @@ class HRX::CLI < Thor
     raise Thor::Error.new(e.message)
   end
 
+  desc "extract <archive>", "Extract files from an archive"
+  option :directory, aliases: :d, type: :string,
+         banner: "Directory to which files are extracted"
+  long_desc <<-LONGDESC
+    Extract the files from an archive.
+
+    This puts the files in the directory given by the --directory option, which
+    defaults to the name of the archive without ".hrx".
+
+    > hrx extract test/specs.hrx
+
+    Extracted 12 files to test/specs.
+  LONGDESC
+  def extract(archive)
+    directory = options[:directory] || begin
+        unless archive.end_with?(".hrx")
+          raise Thor::Error.new(
+            "Can't determine a default output directory for #{archive}. Pass one " +
+            "with --directory.")
+        end
+        directory = archive[0...-".hrx".length]
+      end
```

Comment:
```suggestion
    end
```
It feels like this `end` should line up with the beginning of `directory =` at line 58. It's a little unclear where the block beginning line 58 ends otherwise.

Ran both ways through Rubocop (with default rule set) and Rubocop didn't complain either way.